src/patternfly/components/Select/select.scss

```diff
@@ -60,6 +60,7 @@
   --pf-c-select__toggle-arrow--MarginLeft: var(--pf-global--spacer--md);
   --pf-c-select__toggle-arrow--MarginRight: var(--pf-global--spacer--sm);
   --pf-c-select__toggle-arrow--with-clear--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-select__toggle-arrow--m-top--m-expanded__toggle-arrow--Transform: rotate(-180deg);
```

Comment:
@mattnolting should we make this a positive number for consistency, or does it matter?

Reply:
That sounds right, good catch!